todo_app/app.py

```diff
@@ -92,10 +111,14 @@ def index_get():
     @app.route('/delete', methods=["POST"])
     @login_required
     def delete_item():
+        
         if users[str(current_user.get_id())].isWriter():
             val = request.form.get("deletebutton")
+            mylog.debug('User:' + users[str(current_user.get_id())].name + ' is trying to delete  item ' + val)
             session_items.delete_item(db,val)
             return redirect(url_for('index_get'))
+        else:    
+            mylog.error('User:' + users[str(current_user.get_id())].name + ' is trying to delete  item but it is now authorized.')
```

Comment:
Really minor, but would `they are not authorized` read better than `it is now authorized`?